src/pages/Detail/Detail.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import Star from './Components/Star';
+import Review from './Components/Review';
+import deliveryFee from './배송정보.jpg';
+import './Detail.scss';
+
+const Detail = () => {
+  const params = useParams();
+  const productId = params.id;
```

Comment:
이렇게도 동작하지만, 구조분해할당을 활용하면 좀 더 깔끔하게 작성할 수도 있습니다!
```suggestion
  const { id } = useParams();
```
혹은, productId로 사용하시려면
```suggestion
  const { id: productId } = useParams();
```
이렇게 작성할 수 있습니다!